admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,542 @@
+
+[[determine-default-user-roles]]
+== Determining Which Roles Users Get by Default
+
+When a user first logs in, there is a default set of permissions that is
+applied to that user. The scope of permissions that a user can have is
+controlled by the
+link:../architecture/additional_concepts/authorization.html#roles[various types]
+of
+link:../admin_guide/manage_authorization_policy.html[roles within OpenShift]:
+
+- `ClusterRoles`
+- `ClusterRoleBindings`
+- `Roles` (project-scoped)
+- `RoleBindings` (project-scoped)
+
+You may want to modify the default set of permissions. In order to do this, it's important to understand the default groups and roles assigned, and to be aware of the roles and users bound to
+link:../admin_solutions/user_role_mgmt.html#view-roles-users-project[each project] or
+link:../admin_solutions/user_role_mgmt.html#view-roles-users-cluster[the entire cluster].
+
+=== Leveraging Default Groups
+
+There are special groups that are assigned to users. You can target users with these groups, but you cannot modify them. These special groups are as follows:
+
+[options="header"]
+|===
+
+|Group |Description
+
+|`system:authenticated`
+|This is assigned to all users who are identifiable to the API. Everyone who is not `system:anonymous` (the user) is in this group.
+
+|`system:authenticated:oauth`
+|This is assigned to all users who have identified using an oauth token issued by the embedded oauth server. This is not applied to service accounts (they use service account tokens), or certificate users.
+
+|`system:unauthenticated`
+|This is assigned to users who have not presented credentials. Invalid credentials are rejected with a 401 error, so this is specifically users who did not try to authenticate at all.
+|===
+
+You may find it helpful to target users with the special groups listed above.
+For example, you could share a template with all users by granting
+`system:authenticated` access to the template.
+
+The "default" permissions of users are defined by which roles are bound to the
+`system:authenticated` and `sytem:authenticated:oauth` groups. As mentioned
+above, you are not able to modify membership to these groups, but you can
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[change the roles bound to these groups].
+For example, to bind a role to the `system:authenticated` group for all projects
+in the cluster:
+
+----
+$ oadm policy add-cluster-role-to-group <role> system:authenticated
+----
+
+Currently, by default the `system:authenticated` and `sytem:authenticated:oauth`
+groups receive the following roles:
+
+[options="header"]
+|===
+
+|Role |Description
+
+|`shared-resource-viewer`
+|For the `openshift` project: allows users to see templates and pull images.
+
+|`basic-user`
+|For the the entire cluster: allows users to see their own account, check for information about requesting projects, see which projects they can view, and check their own permissions.
```

Comment:
I don't think a colon works in the description above. A full stop should do. There's one below as well.